client/src/app/[locale]/(admin)/admin/organizations/[id]/requests-id.tsx

```diff
@@ -0,0 +1,272 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import { useTranslations } from 'next-intl';
+import { FieldArray, Form, Formik, FormikErrors, FormikValues } from 'formik';
+
+import {
+  Button,
+  SmallBtn,
+  DateField,
+  Accordion,
+  FileField,
+  InputField,
+  ButtonIcon,
+  ModalAdmin,
+  showMessage,
+  ModalContent,
+  organizationValidation,
+  organizationInitialValues,
+} from '@/components';
+
+const RequestsId = () => {
+  const router = useRouter();
+  const btn = useTranslations('button');
+  const text = useTranslations('inputs');
+  const modal = useTranslations('modal');
+  const error = useTranslations('validation');
+
+  const [isOpenSave, setIsOpenSave] = useState<boolean>(false);
+  const [isOpenAccept, setIsOpenAccept] = useState<boolean>(false);
+  const [isOpenDecline, setIsOpenDecline] = useState<boolean>(false);
+
+  const onSubmit = (values: FormikValues) => {
+    console.log('data', values);
```

Comment:
Should be removed because it exposes PII (Personal Identity Identifiable) data

Reply:
Will be deleted when there is a query to the database